README.md

```diff
@@ -2,10 +2,14 @@
 In this project, I analyzed a dataset and then communicated my findings about it. I used the Microsoft Excel, My SQL to make the analysis easier. This data set contains of two different sheet (The 'tmbd_5000_credits' and 'tmbd_5000_movies) containing information about 4,800 movies collected from The Movie Database (TMDb), including user budget and revenue Certain columns, like ‘cast’, 'crew and ‘genres’, contain multiple values separated by pipe (:) characters. The two columns budget and revenue shows the associated movie in terms of 2010 dollars, accounting for inflation over time
 # What do i need to install? You will need an installation of My SQL and the following libraries
 ~Microsoft Excel
+
 ~Tableau
 # What question i will be exploring?
 ~Do movies with high popularity achieve high revenue?
+
 ~What are the most filmed genres in this whole dataset?
+
 ~Is there a correlation between a movie budget and its revenue?
 # What have i learned?
 
```

Comment:
After completing the project, I learned the following

Reply:
- All the steps involved in a typical data analysis process
- Being comfortable posing questions that can be answered with a given dataset and then answering those questions
- Knowing how to investigate problems in a dataset and wrangle the data into a format I can use
- Having practice communicating the results of my analysis